Mathlib/Algebra/Hom/Commute.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2018 Patrick Massot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Massot, Kevin Buzzard, Scott Morrison, Johan Commelin, Chris Hughes,
+  Johannes Hölzl, Yury Kudryashov
+-/
```

Comment:
We're not very consistent about this, but I'm starting to encourage people to write `Ported by: <name>` here.